local student performance dashboard/StudentDashboard.js

```
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import './StudentDashboard.css'; // import the CSS file for styling
+
+const StudentDashboard = () => {
+  const [studentData, setStudentData] = useState(null);
+
+  useEffect(() => {
+    try {
+      const storedData = localStorage.getItem('studentData');
+      if (storedData) {
+        const data = JSON.parse(storedData);
+        if (data && data.studentId && data.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)